packages/usa-validation/src/index.js

```diff
@@ -1,14 +1,44 @@
 const behavior = require("../../uswds-core/src/js/utils/behavior");
 const validate = require("../../uswds-core/src/js/utils/validate-input");
+const { prefix: PREFIX } = require("../../uswds-core/src/js/config");
+const selectOrMatches = require("../../uswds-core/src/js/utils/select-or-matches");
+
+const VALIDATE_INPUT = "input[data-validation-element]";
+const CHECKLIST_ITEM = `.${PREFIX}-checklist__item`;
 
 function change() {
   validate(this);
 }
 
-const validator = behavior({
-  "keyup change": {
-    "input[data-validation-element]": change,
+const validator = behavior(
+  {
+    "input change": {
+      [VALIDATE_INPUT]: change,
+    },
   },
-});
+  {
+    init(root) {
+      selectOrMatches(VALIDATE_INPUT, root).forEach((item) => {
+        const validationParent = item.parentNode;
+        const checklistItems =
+          validationParent.querySelectorAll(CHECKLIST_ITEM);
+
+        item.setAttribute("aria-controls", "validate-code");
+        item.setAttribute("aria-live", "polite");
+        item.setAttribute("aria-atomic", true);
```

Comment:
It might be better to move these scaffolding lines to its own function like `enhanceValidation`. As well as export them after init.

See example from accordion's JS

https://github.com/uswds/uswds/blob/8d2c7716890606e342877e916d1622eb8f1eaa0b/packages/usa-accordion/src/index.js#L84-L95

Reply:
I created an enhanceValidation() function that sets up all the aria pieces. I admit that I might not fully grasp the concept, so please reach out if this needs revision.